assets/timber.scss.liquid

```diff
@@ -113,6 +113,9 @@ $colorNavText: #333;
 $colorFooterBg: {{ settings.color_footer_bg }};
 $colorFooterText: {{ settings.color_footer_text }};
 
+// Logo max width
+$logoMaxWidth: {{ settings.logo_max_width | default: '450' }}px;
```

Comment:
That is smart! @montalvomiguelo are you able to adjust that line and we can merge it in.